src/atomic.py

```diff
@@ -79,6 +79,10 @@ def look_at_point(x_node, y_node, z_node):
 
 def look_at_face_point(x, y, z):
 	evl.look_at_point(x, y, z)
+	#evl.gaze_at_point(x, y, z)
```

Comment:
Hi Linas,
 I verified that calling blender set_face_target sets bot the head and the gaze, while set_gaze_target sets only the gaze